gradle.properties

```diff
@@ -18,14 +18,14 @@ group=com.trevjonez.composer
 
 CGP_VERSION=1.0.0-rc09
 
-AGP_VERSION=4.0.1
+AGP_VERSION=4.1.0
```

Comment:
Had to bump due to an error (haven't investigate what is actually a reason, probably older plugin doesn't support some of the gradle apis)

> * What went wrong:
Execution failed for task ':commander:testSupport:compileNewerDebugJavaWithJavac'.
> Failed to query the value of task ':commander:testSupport:compileNewerDebugJavaWithJavac' property 'options.generatedSourceOutputDirectory'.
   > Querying the mapped value of map(java.io.File property(org.gradle.api.file.Directory, fixed(class org.gradle.api.internal.file.DefaultFilePropertyFactory$FixedDirectory, /Users/runner/work/composer-gradle-plugin/composer-gradle-plugin/commander/testSupport/build/generated/ap_generated_sources/newerDebug/out)) org.gradle.api.internal.file.DefaultFilePropertyFactory$ToFileTransformer@21eb3dc0) before task ':commander:testSupport:compileNewerDebugJavaWithJavac' has completed is not supported
   

https://developer.android.com/studio/releases/gradle-plugin#4-1-0